developer/src/kmcmplib/tests/fixtures/valid-keyboards/compile_legacy.bat

```diff
@@ -1,4 +1,7 @@
 @echo off
 echo Compiles the keyboards using the legacy kmcomp.exe
 echo to use as baseline comparisons for kmcmplib
+rem TODO: we'll need a binary download for kmcomp.exe
```

Comment:
Isn't this `kmc.exe` now?

Reply:
This is to compare kmc.js to kmcomp.exe, to verify that behaviour hasn't diverged.